tools/alive.cpp

```diff
@@ -19,19 +20,21 @@ using namespace std;
 
 
 static void show_help() {
-  cerr <<
-    "Usage: alive2 <options> <files.opt>\n"
-    "Options:\n"
-    " -root-only\t\tCheck the expression's root only\n"
-    " -v\t\t\tVerbose mode\n"
-    " -smt-stats\t\tShow SMT statistics\n"
-    " -smt-to:x\t\tTimeout for SMT queries in ms\n"
-    " -max-mem:x\t\tMax memory consumption in MB (aprox)\n"
-    " -smt-verbose\t\tPrint all SMT queries\n"
-    " -skip-smt\t\tSkip all SMT queries\n"
-    " -disable-poison-input\tAssume input variables can never be poison\n"
-    " -disable-undef-input\tAssume input variables can never be undef\n"
-    " -h / --help\t\tShow this help\n";
+  cerr << "Usage: alive2 <options> <files.opt>\n"
+          "version "
+       << alive_version
```

Comment:
It is a really good idea to use the C string here, not a macro, so that version change only causes relinking but not recompilation.